docs/flashbots-core/releases/v0.2-release.mdx

```diff
@@ -0,0 +1,163 @@
+s_{v0.1} = \frac{\Delta_{coinbase} + \sum_{T\in U}g_Tp_T}{\sum_{T\in U}g_T}
+$$
+</TabItem>
+<TabItem value="v0.2-pre">
+
+$$
+s_{v0.2-pre} = \frac{\Delta_{coinbase}}{\sum_{T\in U}g_T}
+$$
+</TabItem>
+<TabItem value="v0.2">
+
+$$
+s_{v0.2} = \frac{\Delta_{coinbase} + \sum_{T\in U}g_Tp_T - \sum_{T\in M \cap U}g_Tp_T}{\sum_{T\in U}g_T}
+$$
+</TabItem>
+</Tabs>
+
+$s$: bundle $U$ _score_ used to sort bundles.  
+$U$: ordered list of transactions $T$ in a bundle.  
+$M$: set of transactions $T$ in the mempool.  
+$g_{T}$: _gas used_ by transaction $T$.  
+$p_{T}$: _gas price_ of transaction $T$.  
+$\Delta_{coinbase}$: coinbase difference from direct payment.  
+  
+#### Bundle validation
+The relay removes the temporary limitation of two 'from' addresses per bundle. The bundle can once again contain transactions from an arbitrary number of EOAs.
+
+#### Bundle delivery
+The `eth_sendBundle` RPC method on mev-geth is put into maintenance mode in the v0.2 release. Instead, mev-geth establishes an outgoing websockets connection with the relay and streams bundles as they become available. This improves the security of the miner infrastructure as it no longer requires exposing an incoming RPC port.
+
+To connect to the relay over websockets, each miner will receive an access key which they must add to their mev-geth node to perform the authentication.
+
+#### Block construction
+A new parameter `revertingTxHashes` is added to the bundles which allows the searcher to list the transactions in their bundle which are allowed to revert. By default, all transactions included in a bundle should never revert on chain.
+
+A [simple bundle merging algorithm](https://hackmd.io/@flashbots/core-v2-proposal#Bundle-merging) is introduced to mev-geth which allows multiple bundles to be merged together in a block so long as they do not cause an unexpected revert, or pass a certain amount of gas. The number of bundles mev-geth will attempt to merge together `miner.maxmergedbundles` and the amount of blockspace available for flashbots bundles `miner.maxbundlegas` can be configured by the miner according to the performance of their hardware and their risk tolerance.
+
+#### Strict profit switching
+A new config parameter `miner.strictprofitswitch` is added to mev-geth which allows the miner to force the flashbots workers to wait for the vanila block production to complete before evaluating a flashbots block. This allows the miner to avoid a race condition where they may mine a block which appears less profitable than a vanilla block, but comes at the cost of increasing the number of empty blocks mined. A miner should only enable this feature if they know exactly what they are doing.
+
+## Timeline
+
+**May 6th**: launch v0.2 relay
+- v0.2 relay starts accepting new bundle format
+- searchers are asked to update their bots to send bundles to v0.2 relay endpoint
+- v0.2 bundles are converted to v0.1 bundles in the relay and submitted to v0.1 miners along with v0.1 bundles
```

Comment:
Does this conversion entail anything other than dropping the `revertingTxHashes` field? Or what is meant here?